src/app/attendance/components/AttendanceFormBuilder.tsx

```diff
@@ -162,112 +165,123 @@ const AttendanceFormBuilder = () => {
-                    <SelectValue placeholder="Select a user">
-                      {field.value
-                        ? roleBaseMembers.find((m) => m.id === field.value)
-                            ?.name
-                        : "Select a user"}
-                    </SelectValue>
-                  </SelectTrigger>
-                  <SelectContent>
-                    {roleBaseMembers.map((user) => (
-                      <SelectItem key={user.id} value={user.id.toString()}>
-                        {user.name}
-                      </SelectItem>
-                    ))}
-                  </SelectContent>
-                </Select>
-              </FormControl>
-              <FormMessage />
-            </FormItem>
-          )}
-        />
+                    <SelectTrigger
+                      className={`border rounded-md p-2 ${
+                        form.formState.errors.memberId
+                          ? "border-red-500"
+                          : "border-gray-300"
+                      }`}
+                    >
+                      <SelectValue placeholder="Select a user">
+                        {field.value
+                          ? roleBaseMembers.find((m) => m.id === field.value)
+                              ?.name
+                          : "Select a user"}
+                      </SelectValue>
+                    </SelectTrigger>
+                    <SelectContent>
+                      {roleBaseMembers.map((user) => (
+                        <SelectItem key={user.id} value={user.id.toString()}>
+                          {user.name}
+                        </SelectItem>
+                      ))}
+                    </SelectContent>
+                  </Select>
+                </FormControl>
+                <FormMessage />
+              </FormItem>
+            )}
+          />
 
-        <FormField
-          control={form.control}
-          name="startTime"
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel>Start Time</FormLabel>
-              <FormControl>
-                <Input
-                  type="time"
-                  {...field}
-                  onClick={(event) => event.currentTarget.showPicker()}
-                  className={`border rounded-md p-2 ${
-                    form.formState.errors.startTime
-                      ? "border-red-500"
-                      : "border-gray-300"
-                  }`}
-                />
-              </FormControl>
-              <FormMessage />
-            </FormItem>
-          )}
-        />
+          <FormField
+            control={form.control}
+            name="startTime"
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Start Time</FormLabel>
+                <FormControl>
+                  <Input
+                    type="time"
+                    {...field}
+                    onClick={(event) => event.currentTarget.showPicker()}
```

Comment:
What's going on here?

Reply:
When you click on this input, showPicker() will open the date picker without requiring the user to click the calendar icon.